### PR TITLE
Cloudy Stay Apart

### PR DESCRIPTION
**Pull request by trieloff**

### CODE REVIEW

**Comment by stefan-guggisberg** [comment]
Oct 20 is a company PTO ...

**Comment by kptdobe** [comment]
> Oct 20 is a company PTO ...

Oct 20 was not but Nov 20 is but in fact it is Nov 24 :)

**Review by rofe — APPROVED** [review]

**Comment by rofe** [comment]
Why is https://github.com/adobe/helix-home/pull/163/commits/f8b733dbcc72973ef13e072e2df9ee41e447a106 part of this PR? 😄

File: togetherapart/2-cloudy.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+5. Talk to anyone at any time
+6. Use a breakout Bluejeans session for discussions and list it [below](#breakouts)
+7. No recordings will be made
+8. If you need a break, take a break
+
+### Location
+
+> Where is this going to happen? Do you have a windowless conference room blocked out?
+
+[https://bluejeans.com/743383165](https://bluejeans.com/743383165)
```
Comment (rofe) [inline togetherapart/2-cloudy.md:48]: https://bluejeans.com/344896245

**Review by rofe — APPROVED** [review]

**Pull request merged**